src/services/resource-manager.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+
+'use strict';
+
+import {logFactory} from '../platform/log-web.js';
+
+const log = logFactory('resource-mgr');
+
+export type Reference = number;
+
+// TODO(sjmiles): demonstrate simple concept for tracking objects across the PEC
+
+/**
+ * Allows developer to reference + use resource without passing them between the `Services` bus.
+ * @see src/service.ts
+ */
+export class ResourceManager {
+
+  static references = [];
```

Comment:
type?

Reply:
This should be a list of `any`. So I omitted the type.

Reply:
7d3c7aa